src/main/webapp/portal/notifications/partials/notifications-full.html

```diff
@@ -1,22 +1,24 @@
 <div class="portlet-frame"  ng-controller="NotificationController as notificationCtrl">
 	<portlet-header app-title="Notifications" app-image="img/square.jpg" app-description="Get Notified of important issues."></portlet-header>
 
-	<div class="portlet-body">
+	<div class="portlet-body notifications">
 		<div class="inner-nav-container">
 			<ul class="inner-nav">
 				<li ng-class="{ active : mode === 'new' }"><a href='javascript:;' ng-click="switch('new')"  aria-label='switch to see new notifications'>New</a></li>
-				<li  ng-class="{ active : mode === 'old' }"><a href='javascript:;' ng-click="switch('old')" aria-label='switch to see old and dismissed notifications'>Old/Dismissed</a></li>
+				<li  ng-class="{ active : mode === 'old' }"><a href='javascript:;' ng-click="switch('old')" aria-label='switch to see old and dismissed notifications'>Dismissed</a></li>
 			</ul>
 		</div>
 		<div>
 			<ul class="notifications-list">
           <li class="notification-item" ng-repeat="notification in notifications" ng-if='shouldShow(notification)'>
-            <a ng-href='{{notification.actionURL}}' target='_blank' aria-label='{{notification.actionAlt}}'>{{ notification.title }}</a>
-            <button class='btn btn-flat' ng-click='dismiss(notification)' ng-hide="!notification.dismissable || isDismissed(notification)">
-              <i class='fa fa-trash'></i>
-            </button>
+						<a class="content" ng-href='{{notification.actionURL}}' target='_blank' aria-label='{{notification.actionAlt}}'>
+	            <p>{{ notification.title }}</p>
+						</a>
+						<div class="action" ng-click='dismiss(notification)' ng-hide="!notification.dismissable || isDismissed(notification)">
+	            <i class='fa fa-times'tooltip="Dismiss notification" tooltip-placement="top" tooltip-popup-delay="300" tooltip-trigger="mouseenter"></i>
+						</div>
```

Comment:
Ugh why don't text editors just display the same way? Atom seems messed up for some reason...using Sublime to fix